DashAI/front/src/components/experiments/RunnerDialog.jsx

```diff
@@ -32,15 +32,17 @@ function RunnerDialog({ experiment, expRunning, setExpRunning }) {
     setLoading(true);
     try {
       const runs = await getRunsRequest(experiment.id.toString());
-      const runInExecution = runs.find((run) => run.status === 2); // search a run with status "running"
+      const runInExecution = runs.find((run) => run.status === 2); // searches for a run with the status "running"
```

Comment:
```suggestion
      const runsInExecution = runs.find((run) => run.status === 2); // searches for a run with the status "running"
```

Reply:
That line searches if there is any run currently running.

I used "run" instead of "runs" because the `find` method returns the first element in the array that satisfies the condition.

Reply:
Then, maybe call this variable firstRunInExecution``?